### ISSUE TITLE
Type error: MYSQL, Can't serialize value / Expected type Int but was Long

### ISSUE THREAD

**Issue by Rivi-IT** [dbeaver/cloudbeaver#206]
**Describe the bug**
We are looking for a cloud based SQL Management Tool - so we take a first look at Cloudbeaver.
The installation is container based with podman.
- System OS = CentOS 8 minimal
- DBM System is MariaDB 10.x
- Database is the WordPress database.

**To Reproduce**
Login to CloudBeaver as Admin or User
Connection to SQL Server as root or User with grant all for WordPress database 
DB Admin with grant all for database <db_name>.
Select table "wp_options", try to open table "wp_options".

**Error**
 Can't serialize value (/readDataFromContainer/results[0]/resultSet/columns[2]/maxLength) : Expected type 'Int' but was 'Long'.

**Screenshots**
![CloudBeaver-Error-BigInt](https://user-images.githubusercontent.com/67333044/98264126-e05a6a80-1f87-11eb-9bbb-97a8fa90853e.PNG)

![CloudBeaver-Error-Fields](https://user-images.githubusercontent.com/67333044/98264045-c9b41380-1f87-11eb-89bd-06884af2a0dc.PNG)

**Desktop (please complete the following information):**
 - OS: Win 10
 - Browser: Firefox 
 - Version: 82 
 - Browser: Vivaldi
 - Version: 3.3.2022

 - OS: Lubuntu 20.04
 - Browser: Firefox 
 - Version: 82

**Additional context**

**Comment by kseniiaguzeeva** [comment]
Thank you for the bug report.

**Comment by serge-rider** [comment]
Fixed.
GraphQL (java-graphql library) doesn't allow Int values bigger than maximum 32bit signed integer.
Also JS and GraphQL don't support 64 bit integers (as JS represents all numbers as double precision float values).

I've changed GQL data types from Int to Float in metadata describing data types. It fixed this particular issue.

However we may have other problems with 64 bit long integers in future.
